### ISSUE TITLE
Create component - Link

### ISSUE THREAD

**Issue by B3nz01d** [Orange-OpenSource/ouds-flutter#46]
# Description
**As a** user of the Ouds library
**I want** to be able to use the component
**So that** I can add an Ouds component in my own projects

**As a** user of the library
**I want** to be able to use the component
**So that** I have access to the documentation on how to use it

**As a** user of the Demo App
**I want** to access the Component 
**So that** I can understand how to use it:
* while using the demo app I can interact with the available options and observe the changes on the component itself
* I can see the code associated to the component customization
* I can copy and paste the code from the app based on the selected customization
* or when looking into the code of the app itself, I would also understand how to use it

# Context 
The component is described in the Design tools and allows to describe the various states and variants.
## Variants
TBD

## Demo App
TBD

**Comment by nouha06** [comment]
📣 New Firebase App Distribution Alpha upload available 🚀 (Android)

Please, get the new build from App Tester app, or contact the project maintainers (@Tayebsed93) to get access.

Display name: DesignToolbox (alpha) (441)
Version: 0.6.0
Build type: Alpha
Build details (GitHub): 405
cc @B3nz01d (product owner)
cc @Tayebsed93 @nouha06 @AhmedAmineZr (dev team)
cc @jerome-regnier (design team)
cc @pya35 (accessibility team)

**Comment by nouha06** [comment]
📣 New TestFlight Alpha upload available 🚀 (iOS)

Please, get the new build from TestFlight app, or contact the project maintainers (@Tayebsed93) to get access.

Display name: Design Toolbox Flutter (441)
Version: 0.6.0
Build type: Alpha
Build details (GitHub): 405
cc @B3nz01d (product owner)
cc @Tayebsed93 @AhmedAmineZr @nouha06 (dev team)
cc @jerome-regnier (design team)
cc @pya35 (accessibility team)

**Comment by jerome-regnier** [comment]
Bonjour @nouha06 ,

Review Figma : 
https://www.figma.com/design/yKtkEY1DOl7YDvmt7jq0QF/-OUDS-Core-Lib--Review?node-id=69580-983&t=8zL96TM0EOkwIrl9-1

Sur le link next en small, il y a un décalage entre le text et l'icône chevron. Si le token utilisé est le bon, il faudrait regarder si tu as la bonne icône car il y aurait un décalage dans celle que tu utilises.

**Comment by nouha06** [comment]
Hello @pya35 

I wanted to follow up and inquire if you have had the opportunity to review the Link component.

Thank you in advance for your feedback.

**Comment by pya35** [comment]
> Hello [@pya35](https://github.com/pya35)
> 
> I wanted to follow up and inquire if you have had the opportunity to review the Link component.
> 
> Thank you in advance for your feedback.

@nouha06 I've attached 2 issues, I'm surprised that the basics like a simple link is not conform. I don't know the dev envrionment for Flutter, are you able to use an a11y linter as in Android Studio or Accessibility Inspector on MacOS?

**Comment by nouha06** [comment]
The issue has been fixed and the ticket has been closed, as we cannot release a new version. That’s why the TestFlight blocking label was applied.

**Comment by B3nz01d** [comment]
Closed during Sprint # 23

**Comment by nouha06** [comment]
📣 New Firebase App Distribution BETA upload available 🚀 (Android)

Please, get the new build from App Tester app, or contact the project maintainers (@Tayebsed93) to get access.

Display name: DesignToolbox (BETA) (485)
Version: 0.6.0
Build type: BETA
Build details (GitHub): 405
cc @B3nz01d (product owner)
cc @Tayebsed93 @AhmedAmineZr @nouha06 (dev team)
cc @jerome-regnier (design team)
cc @pya35 (accessibility team)

**Comment by nouha06** [comment]
New TestFlight BETA upload available 🚀 (iOS)

Please, get the new build from TestFlight app, or contact the project maintainers (@Tayebsed93) to get access.

Display name: Design Toolbox Flutter (485)
Version: 0.6.0
Build type: BETA
Build details (GitHub): 405
cc @B3nz01d (product owner)
cc @Tayebsed93 @AhmedAmineZr @nouha06 (dev team)
cc @jerome-regnier (design team)
cc @pya35 (accessibility team)
